docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+services:
+   database:
+     image: martlark/pg_dump:12.1
```

Comment:
probably should make this a more recent version of postgres then I'm lazily using at home.

Reply:
I'm ashamed to have not upgraded!

Reply:
Did you want me to upgrade this? Not sure what else you might be looking for prior to the merge